dtfs-central-api/src/v1/controllers/tfm/deal/tfm-get-deals.controller.js

```diff
@@ -69,23 +70,26 @@ const findDeals = async (searchString, sortBy, fieldQueries, callback) => {
       dateString = String(moment(date).format('DD-MM-YYYY'));
     }
 
+    const searchStringRegex = escapeRegExp(searchString);
```

Comment:
Just a suggestion that maybe variable name `searchStringRegex` is not clear, because we don't expect user to enter regex.

Maybe `safeSearchString` or `searchStringEscaped` is better?